src/test/resources/expected_helm_output/jira/output.yaml

```diff
@@ -43,11 +43,19 @@ metadata:
     app.kubernetes.io/version: "9.4.6"
     app.kubernetes.io/managed-by: Helm
 data:
-  jmx-config.yaml: |
-    lowercaseOutputLabelNames: true
-    lowercaseOutputName: true
+  jmx-config.yaml: |  
+    ---
+    startDelaySeconds: 0
     rules:
-      - pattern: ".*"
+      - pattern: '(java.lang)<type=(\w+)><>(\w+):'
+        name: java_lang_$2_$3
+      - pattern: 'java.lang<type=Memory><HeapMemoryUsage>(\w+)'
+        name: java_lang_Memory_HeapMemoryUsage_$1
+      - pattern: 'java.lang<name=G1 (\w+) Generation, type=GarbageCollector><>(\w+)'
+        name: java_lang_G1_$1_Generation_$2
+      - pattern: '.*'
+    blacklistObjectNames:
+      - 'com.atlassian.jira:connectionpool=connections,connection=*,name=BasicDataSource'
```

Comment:
Why is this hidden?

Do we want to add some comments regarding specific rules so we have the decisions and reasons documented?